perfmetrics/scripts/testing_on_gke/examples/run-gke-tests.sh

```diff
@@ -565,6 +558,12 @@ function createCustomCsiDriverIfNeeded() {
     make generate-spec-yaml
     printf "\nBuilding a new custom CSI driver using the above GCSFuse binary ...\n\n"
     registry=gcr.io/${project_id}/${USER}/${cluster_name}
+    if ! which uuidgen; then
+      # try to install uuidgen
```

Comment:
Should we move this to installDependencies section?

Reply:
That was my first thought, but this utility isn't needed unless user wants to build custom driver. Hence, I have kept it here to avoid unnecessary installation/checking/log.

In the future, I'll moving around more of these installations to wherever they are needed.